devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/client.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/http;
+
+// Global HTTP Client
```

Comment:
@Ibaqu Shall we get rid of comments from places where it's self explanatory?

Reply:
removed comments